.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,11 @@
+name: Auto approve
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2.0.0
+      if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
+      with:
+        github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
How secrets would work in this case? is it a personal token? or shared token for this project (via settings)?

Reply:
@jandrade It seems like it's auto-configured! More details here: https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables

Reply:
As for secrets in general for actions, they are specified on the repo. Once set, you cannot see their values.